src/sentry/static/sentry/app/components/forms/selectControlLegacy.tsx

```diff
@@ -88,6 +91,13 @@ class SelectControlLegacy extends React.Component<Props> {
         typeof choices === 'function' ? choices(this.props) : choices
       ) || options;
 
+    const isMulti = props.multiple || props.multi;
+
+    // HACK(mgaeta): ReactSelect isn't reading `default` correctly so we're using `value`.
```

Comment:
@mgaeta instead of hacking it here, we should make sure we don't use the legacy select control